packages/appeals-service-api/src/schemas/lpa.js

```diff
@@ -74,11 +73,14 @@ module.exports = class LPA {
     return (await csvParser(lpaList))
       .filter(({ LPA19CD }) => {
         /* Filter out Scotland and Northern Ireland LPAs */
-        const include = LPA19CD.startsWith('S') === false && LPA19CD.startsWith('N') === false;
+        const include =
+          LPA19CD.startsWith('S') === false &&
+          LPA19CD.startsWith('N') === false &&
+          LPA19CD.startsWith('W') === false;
         logger.trace({ include, id: LPA19CD }, 'Filtering out Scottish and Northern Irish LPAs');
```

Comment:
and I'm guessing 'W'elsh?

Reply:
It is.  I will extend that comment.  Seems the original file contained Welsh lpa as well, but this new file only contains English ones.